lib/licenses.nix

```diff
@@ -1261,11 +1261,21 @@ in mkLicense lset) ({
     fullName = "xinetd License";
   };
 
+  xskat = {
```

Comment:
Split this transposition to its own commit.

Reply:
What is the most efficent way to split it up into an extra commit?

Reply:
You can use git reset HEAD^ or git undo from git-extras